src/main/kotlin/app/revanced/patches/youtube/video/speed/remember/RememberPlaybackSpeedPatch.kt

```diff
@@ -1,33 +1,33 @@
-package app.revanced.patches.youtube.video.speed.remember.patch
+package app.revanced.patches.youtube.video.speed.remember
 
 import app.revanced.extensions.exception
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstructionsWithLabels
 import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
 import app.revanced.patcher.patch.BytecodePatch
-import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotation.Patch
 import app.revanced.patcher.util.smali.ExternalLabel
 import app.revanced.patches.shared.settings.preference.impl.ArrayResource
 import app.revanced.patches.shared.settings.preference.impl.ListPreference
 import app.revanced.patches.shared.settings.preference.impl.StringResource
 import app.revanced.patches.shared.settings.preference.impl.SwitchPreference
-import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
-import app.revanced.patches.youtube.misc.settings.bytecode.patch.SettingsPatch
-import app.revanced.patches.youtube.video.information.patch.VideoInformationPatch
-import app.revanced.patches.youtube.video.speed.custom.patch.CustomPlaybackSpeedPatch
+import app.revanced.patches.youtube.misc.integrations.IntegrationsPatch
+import app.revanced.patches.youtube.misc.settings.SettingsPatch
+import app.revanced.patches.youtube.video.information.VideoInformationPatch
+import app.revanced.patches.youtube.video.speed.custom.CustomPlaybackSpeedPatch
 import app.revanced.patches.youtube.video.speed.remember.fingerprint.InitializePlaybackSpeedValuesFingerprint
 import com.android.tools.smali.dexlib2.iface.instruction.ReferenceInstruction
 
-@Name("Remember playback speed")
-@Description("Adds the ability to remember the playback speed you chose in the playback speed flyout.")
-@DependsOn([IntegrationsPatch::class, SettingsPatch::class, VideoInformationPatch::class, CustomPlaybackSpeedPatch::class])
-class RememberPlaybackSpeedPatch : BytecodePatch(
-    listOf(
-        InitializePlaybackSpeedValuesFingerprint
-    )
-) {
+@Patch(
+    description = "Adds the ability to remember the playback speed you chose in the playback speed flyout.",
+    dependencies = [IntegrationsPatch::class, SettingsPatch::class, VideoInformationPatch::class, CustomPlaybackSpeedPatch::class]
+)
```

Comment:
```suggestion
@Patch(
    name = "Remember playback speed",
    description = "Adds the ability to remember the playback speed you chose in the playback speed flyout.",
    dependencies = [IntegrationsPatch::class, SettingsPatch::class, VideoInformationPatch::class, CustomPlaybackSpeedPatch::class]
)
```

Reply:
This patch is a dependency of `PlaybackSpeedPatch`